lib/std/os/bits/linux.zig

```diff
@@ -1769,7 +1769,7 @@ pub const rusage = extern struct {
     utime: timeval,
     stime: timeval,
     maxrss: isize,
-    ix_rss: isize,
+    ixrss: isize,
```

Comment:
@data-man I'm not sure why this had an underscore. Do you mind if it's removed?

Reply:
@fengb 
Ok.